frontend/pages/ManageControlsPage/SetupExperience/cards/EndUserAuthentication/components/EndUserExperiencePreview/EndUserExperiencePreview.tsx

```diff
@@ -1,7 +1,12 @@
 import React from "react";
 import classnames from "classnames";
 
-import EndUserAuthEndUserPreview from "../../../../../../../../assets/videos/end-user-auth-preview.mp4";
+import TabNav from "components/TabNav";
+import { Tab, TabList, TabPanel, Tabs } from "react-tabs";
+import TabText from "components/TabText";
+
+import CompanyOwnedEndUserAuthPreview from "../../../../../../../../assets/videos/company-owned-end-user-auth-preview.mp4";
+import PersonalEndUserAuthPreview from "../../../../../../../../assets/images/personal-end-user-auth-preview.png";
```

Comment:
For front-end sync discussion: we can use some Webpack magic to not have to claw up ~7~ 8 subdirectories to get assets.  `import xyz from 'assets/images/someImage.png` would be a lot nicer!

Reply:
i'll look to create a quick eng initiative story